modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -11,3 +13,11 @@
 		}
 	}
 }
+
+.empty-state {
+	left: 39%;
```

Comment:
Well... this seems oddly specific 😂

Reply:
I didn't found something related with empty message in the chart out of the box. I have to position it manually. That makes me sad...

Reply:
FYI https://github.com/recharts/recharts/issues/1581

Reply:
We will change this for a left 50% with translate -50%

Reply:
https://github.com/bryceosterhaus/liferay-portal/pull/70/commits/93121106007125ca3a4747d3952dae739b0e184b